README.md

```diff
@@ -16,16 +16,16 @@ https://github.com/cabal-club/cabal-desktop/releases/
 $ git clone https://github.com/cabal-club/cabal-desktop
 $ cd cabal-desktop
 
-$ npm install             # install dependencies
-$ npm start               # start the application
+$ yarn install             # install dependencies
```

Comment:
Is yarn a requirement now?

Reply:
If not, I think more people know/have npm than yarn, so it might be worth keeping npm here even just for sake of clarity.

Reply:
npm's fine with me! @d4tocchini made some improvements recently that let yarn keep things leaner... but that may only be for doing a production build anyhow. I'll change it back.